server/pulp/plugins/loader/manager.py

```diff
@@ -22,6 +30,43 @@ def __init__(self):
         self.importers = _PluginMap()
         self.profilers = _PluginMap()
         self.catalogers = _PluginMap()
+        self.unit_models = dict()
+
+        # Load the unit models
+        self._load_unit_models()
+
+    def _load_unit_models(self):
+        """"
+        Load all of the Unit Models from the ENTRY_POINT_UNIT_MODELS entry point
```

Comment:
Consider renaming ENTRY_POINT_UNIT_MODELS to PULP_UNIT_MODELS or just UNIT_MODELS as the entry point. It is an entry point so having that in the name makes it longer without adding anything imho.